cg/server/app_config.py

```diff
@@ -0,0 +1,26 @@
+from pydantic_settings import BaseSettings
+
+
+class AppConfig(BaseSettings):
```

Comment:
I think you might need to specify the ENV file you get the values from, right? 

i.e. add a class Config in here that parses the env file:
```suggestion
class AppConfig(BaseSettings):
    ...
    ...
    class Config:
        env_file = str(ENV_FILE)

```

Reply:
BaseSettings takes the variables straight from the environment variables. But if you want to skip exporting them to the environment you can use the example you added to point to a file instead. 

Since the cg container has all variables exported to it I didn't add the Config class since it will not currently be used in production. I think it can easily be added if we decide to point to a file instead. 

Basically my reasoning is. Since we're not using it we can leave it out for now